contrib/src/main/java/com/datatorrent/contrib/cassandra/CassandraPOJOOutputOperator.java

```diff
@@ -172,25 +197,32 @@ public void deactivate()
   @Override
   protected PreparedStatement getUpdateCommand()
```

Comment:
Please add javadocs stating the behaviour in case query is null and if not null..